roles/artemis/templates/application-prod.yml.j2

```
@@ -357,3 +357,36 @@ management:
     metrics:
       export:
         enabled: true
+
+###############################################################################
+# Helios
+###############################################################################
+{% set is_build_agent = (IS_BUILD_AGENT is defined) %}
+
+helios:
+  status:
+    # Disable entirely on build agents
+    enabled: {{ 'false' if is_build_agent else 'true' }}
+    environment-name: ${HELIOS_ENVIRONMENT_NAME:}
```

Where is this evironment variable coming from? This should be:

```suggestion
    environment-name: {{ helios_environment_name }}
```